File: 백준/Bronze/5622. 다이얼/다이얼.py
Dial = ['ABC','DEF','GHI','JKL', 'MNO', 'PQRS', 'TUV', 'WXYZ']
num = input()
time = 0

for i in range(len(num)):
    for j in Dial:
        if num[i] in j:
            time += (Dial.index(j) + 3)
print(time)

File: 백준/Bronze/5622. 다이얼/다이얼.js
var input = require("fs").readFileSync("/dev/stdin").toString().trim();
var phone = {
	2: "ABC",
	3: "DEF",
	4: "GHI",
	5: "JKL",
	6: "MNO",
	7: "PQRS",
	8: "TUV",
	9: "WXYZ",
};
var result = 0;

for (var i = 0; i < input.length; i++) {
	for (var j = 2; j <= 9; j++) {
		if (phone[j].includes(input[i])) {
			result += j + 1;
			break;
		}
	}
}

console.log(result);